BFAIR/FIA_MS/import_statistics.py

```diff
@@ -0,0 +1,123 @@
+import numpy as np
+import pandas as pd
+from pyopenms import FeatureMap, FeatureXMLFile
+
+
+def extractNamesAndIntensities(feature_dir, sample_names, database):
+    """
+    This function take .featureXML files, the output of SmartPeak
+    pre-processing, and extracts the metabolite's reference and it's measured
+    intensity. Furthermore, the peptide references are compared with a
+    database, that has also been used for SmartPeak processing. Only
+    metabolites that appear in the database are being used. Their formula is
+    added to the output dataframe. The names of the files that should be
+    processed are added via a SmartPeak sequence file.
+    Parameters:
+        feature_dir: the path to the directory holding the .featureXML files
+        sample_names: the extracted information from a sequence file
+        database: the organism specific database with the metabolite mapping
+    Returns:
+        extracted_data_all: a dataframe containing the extracted information,
+            the 4 columns include the sample name, the peptide reference for
+            the metabolites, their corresponding formulas and the measured
+            intensity
+    """
+    metabolites_unique = database[0].unique()
+    extracted_data_dict = {}
+    cnt = 0
+    for name in sample_names:
+        features = FeatureMap()
+        FeatureXMLFile().load(
+            feature_dir + "/" + name + ".featureXML", features
+        )
+        for f in features:
+            try:
+                peptideRef = f.getMetaValue("PeptideRef").decode("utf-8")
+            except AttributeError:
+                peptideRef = f.getMetaValue("PeptideRef")
+            if peptideRef in metabolites_unique:
+                formula = database[database[0] == peptideRef][1]
+                extracted_data_dict[cnt] = {
+                    "sample_group_name": name,
+                    "Metabolite": peptideRef,
+                    "Formula": list(formula)[0],
+                    "Intensity": f.getMetaValue("peak_apex_int"),
+                }
+                cnt = cnt + 1
+    extracted_data_all = pd.DataFrame.from_dict(extracted_data_dict, "index")
+    return extracted_data_all
+
+
+def calculateMeanVarRSD(
+    extracted_data_all, sample_name_2_replicate_groups, min_reps=3
+):
+    """
+    The output of the extractNamesAndIntensities() function (or its output
```

Comment:
Same here. The code that Ricardo has committed recently follows the numpy docstrings.  Can you also check if the previous code you have committed conforms to the numpy docstrings standard?